<?xml version="1.0"?>

<launch>
  <!-- camera name prefix -->
  <arg name="cam_name" default="bfly"/>
  <arg name="camera_info_url" default="file://$(find versavis)/cfg/$(arg cam_name).yaml"/>

  <!-- Recieve IMU message. -->
  <node name="relay" pkg="versavis" type="relay" required="true" output="screen">
    <param name="cam_name" value="$(arg cam_name)"/>
    <param name="camera_info_url" value="$(arg camera_info_url)"/>
    <remap from="image_raw" to="$(arg cam_name)/image_raw" />
    <remap from="header" to="$(arg cam_name)/header" />
    <remap from="image_numbered" to="$(arg cam_name)/image_numbered" />
  </node>
</launch>
